### DISCUSSION TITLE
Difference between Aeon vs Sktime?

### DISCUSSION THREAD

**Discussion by jokikim** [aeon-toolkit/aeon#519] [General]
I see alot of similarities between Aeon and Sktime, and noticed some of the scripts still mentioned certain authors of Sktime.

What is the advantage of Aeon over Sktime? Is there a difference in philosophy or direction? Is there deep learning abilities with Aeon?

Thanks,
Jon

**Comment by TonyBagnall** [comment]
hi, 

aeon forked from sktime, see here for my take on what happened. 
https://github.com/aeon-toolkit/aeon/discussions/504
Most core devs moved over to aeon. I cannot speak for those who are now with sktime, you should ask them. For me at least, the aeon priorities are
1. simple docs and general closer alignment with sklearn
2. efficiency a priority over software engineering
3. bespoke implementations a priority over wrapping
4. easy access to sota implementations in time series machine learning, for all areas of application, including forecasting, classification, regression, clustering and annotation
5. reproducible benchmarking experiments of the latest algorithms

We will be including more deep learning, especially since we have @hadifawaz1999 as a new core dev :)

feel free to ask anything here or join our slack

Tony

**Comment by hadifawaz1999** [comment]
Hello @jokikim , as @TonyBagnall mentioned aeon now have much diverse deep learning tasks ! For now we have included all the models from [dl4tsc](https://arxiv.org/pdf/1809.04356.pdf), [InceptionTime](https://arxiv.org/abs/1909.04939) and [H-InceptionTime](https://germain-forestier.info/publis/bigdata2022.pdf) for classification and even started with the regression.

Next plans for aeon is to have more deep learning usability for clustering and forecasting.
